eng/pipelines/templates/scripts/verify-path-length.yml

```diff
@@ -9,7 +9,9 @@ steps:
       scriptSource: inline
       script: |
         # Verifies Length of file path for all files in the SourceDirectory.
-        # Breaks if file path is 260 characters or more
+        # File paths and directory paths must be less than 260 and 248 characters respectively on windows OS
+        # Repo users get 49 characters for the repo clone path
```

Comment:
Can we take in this base path as a template parameter? You can default it to 49 but otherwise it should be configurable instead of hard-coded.

Reply:
I don't understand what you mean, Am not hard coding the base path, am just counting from the root of the repo rather than the whole length of the directory.